### ISSUE TITLE
Pause broken

### ISSUE THREAD

**Issue by tmaly1980** [futurice/react-native-audio-toolkit#67]
I noticed when calling pause() or playPause() that the sound stops but the currentTime keeps incrementing and isPlaying still is true. My player will not pause (at least the time) and I cannot resume playback either because the sound isn't playing.

https://github.com/futurice/react-native-audio-toolkit/blob/master/src/Player.js#L164

Why is this commented out? I uncommented it and now my player is working as expected.

**Comment by pottercomuneo** [comment]
Same problem!! Maybe related to recent upgrate to RN 0.50? It used to work before.

**Comment by xstable** [comment]
This is fixed at the end of 2017...: https://github.com/futurice/react-native-audio-toolkit/commit/231d40a91370a5e7c05c3344e4f8ac0a65e51bc0

Can somebody please bundle version 1.0.7 so that this fix is in current release?

**Comment by xstable** [comment]
Now that RN-Community has take over the Project... can anybody merge this, please ?

**Comment by tmaly1980** [comment]
I actually ended up just switching to react-native-video, which itself supports playing audio much more reliably. But I agree with merging this after a year and a half.

**Comment by xstable** [comment]
Good hint, maybe I also could give it a try to minimize the used components.
But does rn-video also play mp3-Files even like mp4 (only as audio)?

**Comment by tmaly1980** [comment]
Yes, react-native-video plays mp3 files, and it's much better supported than this library.
